src/middleware/account-interventions-middleware.ts

```diff
@@ -0,0 +1,55 @@
+import { NextFunction, Request, Response } from "express";
+import { getNextPathAndUpdateJourney } from "../components/common/constants";
+import { USER_JOURNEY_EVENTS } from "../components/common/state-machine/state-machine";
+import { accountInterventionService } from "../components/account-intervention/account-intervention-service";
+import { ExpressRouteFunc } from "../types";
+import { supportAccountInterventions } from "../config";
+
+export function accountInterventionsMiddleware(
+  service = accountInterventionService()
+): ExpressRouteFunc {
+  return async function (req: Request, res: Response, next: NextFunction) {
+    const email = req.session.user.email.toLowerCase();
+    const { sessionId, clientSessionId, persistentSessionId } = res.locals;
+
+    const accountInterventionsResponse =
```

Comment:
Only call the service when interventions is switched on, so move to the block below?